<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgMultishapeEdition</class>
 <widget class="QDialog" name="CQDlgMultishapeEdition">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edition of Compound Shapes</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>361</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Other properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqBackfaceCulling">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Backface culling</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqWireframe">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Wireframe</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqShowEdges">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Show edges with angle [deg]</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Shading angle [deg]</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqShadingAngle">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>20</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqEdgesAngle">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>41</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqHiddenBorder">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Hidden border</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Colors / texture</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqAdjustColor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust color</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqTexture">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust texture</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqAdjustColor</tabstop>
  <tabstop>qqTexture</tabstop>
  <tabstop>qqShadingAngle</tabstop>
  <tabstop>qqShowEdges</tabstop>
  <tabstop>qqEdgesAngle</tabstop>
  <tabstop>qqHiddenBorder</tabstop>
  <tabstop>qqBackfaceCulling</tabstop>
  <tabstop>qqWireframe</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
